github-actions/start-release/api/github.py

```diff
@@ -11,6 +13,18 @@
     'email': 'root@splunkphantom'
 }
 
+DEFAULT_ENCODING = 'utf-8'
+
+
+def deserialize_text_file(unreleased_md):
+    return base64.b64decode(unreleased_md['content']) \
+        .decode(DEFAULT_ENCODING).split('\n')
+
+
+def handle_http_not_found(ex):
+    if not isinstance(ex, HTTPError) or ex.response.status_code != HTTPStatus.NOT_FOUND:
+        raise ex
```

Comment:
this code doesn't seem to be used anywhere? unless it's used in the unchanged code

Reply:
thanks, seems I don't have plans for this code anymore 😅